### ISSUE TITLE
Skip building Docker image

### ISSUE THREAD

**Issue by shaleenb** [ploomber/soopervisor#103]
Is it possible to skip the step where Soopervisor builds the docker image? I just want to generate the `argo.yaml` file.

**Comment by edublancas** [comment]
Hi, thanks for your feedback!

this isn't possible at the moment, but it makes sense. I'm guessing you wanna take care of the docker building process yourself?

I'll add this to our backlog so we work on it, I'm thinking something like `soopervisor export --skip-docker`

**Comment by shaleenb** [comment]
Yes, that's exactly what I meant. I'd like to build the docker images myself / use an existing docker image. 

Something like `soopervisor export --skip-docker` would be great!

**Comment by edublancas** [comment]
hi @shaleenb, can you give this a try?

please install from git and let us know if this fixes your issue:

```
pip uninstall soopervisor
pip install git+https://github.com/ploomber/soopervisor
```

docs: https://soopervisor.readthedocs.io/en/latest/api/cli.html?highlight=--skip#skip-docker
